sql/hive/src/test/scala/org/apache/spark/sql/hive/execution/HiveDDLSuite.scala

```diff
@@ -372,6 +372,20 @@ class HiveCatalogedDDLSuite extends DDLSuite with TestHiveSingleton with BeforeA
       assert(table.provider == Some("org.apache.spark.sql.hive.orc"))
     }
   }
+
+  test("Create Table LIKE with specified TBLPROPERTIES") {
```

Comment:
can we put all the tests in one suite? There is a TODO to merge `HiveCatalogedDDLSuite` and `HiveDDLSuite`, I think these 2 suites should not have much difference.

Reply:
Which one should be moved to, `HiveCatalogedDDLSuite` or `HiveDDLSuite`?